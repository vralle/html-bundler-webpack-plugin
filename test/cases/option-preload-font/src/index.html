<!DOCTYPE html>
<html>
<head>
  <title>Home</title>
  <link href="./home.scss" rel="stylesheet">
</head>
<body>
  <h1>Home</h1>
  <div>
    <span class="mi">home</span>
    <span class="mi">settings</span>
  </div>
</body>
</html>