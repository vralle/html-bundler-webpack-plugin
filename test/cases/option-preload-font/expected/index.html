<!DOCTYPE html>
<html>
<head>
  <title>Home</title>
  <link rel="preload" href="css/home.bundle.css" as="style">
  <link rel="preload" href="fonts/open-sans-regular.woff2" as="font" type="font/woff2" crossorigin>
  <link rel="preload" href="fonts/MaterialIcons-Regular.woff2" as="font" type="font/woff2" crossorigin>
  <link href="css/home.bundle.css" rel="stylesheet">
</head>
<body>
  <h1>Home</h1>
  <div>
    <span class="mi">home</span>
    <span class="mi">settings</span>
  </div>
</body>
</html>